src/screens/factories/factories.tsx

```diff
@@ -12,6 +12,7 @@ import { UserResourceLink } from "@/entities/user";
 import { PagedResponse } from "@/shared/api";
 import { toast } from "sonner";
 import { formatToTwinfaceDate } from "@/shared/libs";
+import {useRouter} from "next/navigation";
```

Comment:
lets enhance columns:
- factoryBranchesCount
- factoryUsagesCount
- factoryErasersCount
- factoryMultipliersCount
- factoryPipelinesCount

via

```tsx
    cell: (data) => (
      <div className="text-center">{data.getValue<string>()}</div>
    ),
```

Reply:
We don't have text-centering on any other columns, why add it here?